index.html

```diff
@@ -0,0 +1,307 @@
+          >
+            <div
+              class="flex md:flex-col w-[32%] md:w-full gap-2 justify-center items-center"
+            >
+              <img width="44" src="./images/icons2.png" alt="" />
+              <h4 class="text-callAction font-semibold">Forum</h4>
+            </div>
+            <p class="text-secondary w-[68%] md:px-4 md:w-full text-sm">
+              Lorem ipsum dolor sit amet, consectetur adip is icing elit. Vero,
+              iste molestias? Lorem ipsum dolor sit amet.
+            </p>
+          </div>
+          <div
+            class="flex md:flex-col md:items-center gap-4 justify-between bg-gray-500 bg-opacity-20 py-4 rounded my-hover"
+          >
+            <div
+              class="flex md:flex-col w-[32%] md:w-full gap-2 justify-center items-center"
+            >
+              <img width="44" src="./images/icons5.png" alt="" />
+              <h4 class="text-callAction font-semibold">Workshop</h4>
+            </div>
+            <p class="text-secondary w-[68%] md:px-4 md:w-full text-sm">
+              Lorem ipsum dolor sit amet, consectetur adip is icing elit. Vero,
+              iste molestias? Lorem ipsum dolor sit amet.
+            </p>
+          </div>
+          <div
+            class="flex md:flex-col md:items-center gap-4 justify-between bg-gray-500 bg-opacity-20 py-4 rounded my-hover"
+          >
+            <div
+              class="flex md:flex-col md:text-center w-[32%] md:w-full gap-2 justify-center items-center"
+            >
+              <img width="44" src="./images/icons3.png" alt="" />
+              <h4 class="text-callAction font-semibold">CC Ignite</h4>
+            </div>
+            <p class="text-secondary w-[68%] md:px-4 md:w-full text-sm">
+              Lorem ipsum dolor sit amet, consectetur adip is icing elit. Vero,
+              iste molestias? Lorem ipsum dolor sit amet.
+            </p>
+          </div>
+        </div>
+        <div class="flex flex-col items-center">
+          <button
+            class="text-white text-l px-8 py-4 mx-auto bg-callAction md:hidden"
+          >
+            Join CC Global Summit 2015
+          </button>
+          <a
+            href=""
+            class="underline underline-offset-auto hidden uppercase cursor-pointer text-white md:contents"
+            >See the whole program
+          </a>
+        </div>
+      </div>
+    </section>
+
+    <!-- Speakers -->
+    <section id="the-speakers">
+      <div class="container flex flex-col mb-16 mx-auto space-y-8">
+        <div class="flex flex-col gap-4 items-center pt-8 px-2">
+          <h2 class="text-primary text-2xl font-medium">Featured Speakers</h2>
+          <span class="dash bg-callAction"></span>
+        </div>
+        <!-- Speaker -->
+        <div
+          id="speakers"
+          class="flex mx-4 flex-col gap-8 box-border md:grid md:grid-cols-2 md:gap-x-4 md:gap-y-8 md:mx-0"
+        >
```

Comment:
- Kindly, your speaker list photo and description are overlapping and name,descriptions aren’t placed on proper place on the mobile version, please fix it to have a better design👌
👇 given design
![2023-03-16_15-08_1](https://user-images.githubusercontent.com/52010789/225616274-e2a23efd-0b60-4e49-ace1-7399821a711b.png)
👇 your design
![2023-03-16_15-08](https://user-images.githubusercontent.com/52010789/225616533-7f973fd1-8e62-4d9a-af10-b0461579833a.png)